File: 백준/Bronze/28702. FizzBuzz/FizzBuzz.java
import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) throws IOException {
        Scanner sc = new Scanner(System.in);
        String[] input = new String[3];
        for (int i = 0; i < 3; i++) {
            input[i] = sc.next();
        }
        sc.close();
        int answer = 0;
        for (int i = 0; i < 3; i++) {
            if (!input[i].equals("FizzBuzz") && !input[i].equals("Fizz") && !input[i].equals("Buzz")) {
                answer = Integer.parseInt(input[i]) + 3 - i;
                break;
            }
        }
        if (answer % 3 == 0 && answer % 5 == 0) {
            System.out.println("FizzBuzz");
        } else if (answer % 3 == 0) {
            System.out.println("Fizz");
        } else if (answer % 5 == 0) {
            System.out.println("Buzz");
        } else {
            System.out.println(answer);
        }
    }
}

File: 백준/Bronze/28702. FizzBuzz/FizzBuzz.py
numbers = list()
for i in range(3):
    numbers.append(input())
answer = 0

for i in range(3):
    if numbers[i] != "FizzBuzz" and numbers[i] != "Fizz" and numbers[i] != "Buzz":
        answer = int(numbers[i]) + 3 - i
        break

if answer % 3 == 0 and answer % 5 ==0:
    print("FizzBuzz")
elif answer % 3 == 0:
    print("Fizz")
elif answer % 5 == 0:
    print("Buzz")
else:
    print(answer)